styles.css

```diff
@@ -610,3 +626,486 @@ li {
+    letter-spacing: 0.03em;
+    color: #172b4d;
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    justify-content: space-between;
+  }
+
+  .listw li {
+    padding: 8px 12px;
+    background: #ebf0ee;
+    border-radius: 4px;
+  }
+
+  #button {
+    position: relative;
+    left: 1%;
+  }
+
+  /* About section */
+
+  .images_about {
+    display: contents;
+  }
+
+  .img_about1 {
+    position: absolute;
+    width: 288.04px;
+    height: 140.24px;
+    bottom: 55%;
+    left: -13%;
+  }
+
+  .img_about2 {
+    position: absolute;
+    width: 269.55px;
+    height: 244.78px;
+    bottom: 60%;
+    right: -13%;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 80px 0;
+    gap: 48px;
+    position: absolute;
+    width: 1171px;
+    height: 808px;
+    left: 115px;
+    top: 2300px;
+    background: none;
+  }
+
+  .text_about {
+    padding-left: 34%;
+  }
+
+  .text2_about {
+    width: 920px;
+    height: 96px;
+    font-size: 20px;
+    line-height: 32px;
+    padding-top: 15px;
+    position: relative;
+    left: 30px;
+  }
+
+  #button_resume {
+    position: relative;
+    left: 410px;
+    top: -20px;
+  }
+
+  .skills_container {
+    display: inline-grid;
+    align-items: center;
+    padding: 4px 8px;
+    grid-template-columns: auto auto auto;
+    gap: 24px;
+    width: 1171px;
+    height: 360px;
+    padding-top: 0;
+    margin-top: 0;
+    position: absolute;
+    top: 60%;
+    left: 1.5%;
+  }
+
+  .card_skills {
+    align-items: center;
+    justify-content: center;
+    align-content: center;
+    width: 368px;
+    height: 350px;
+    background: #ebf0ee;
+    border: 1px solid #c1c7d0;
+    border-radius: 8px;
+    padding: 4px;
+    margin: 2px;
+  }
+
+  #icons {
+    position: relative;
+    width: 48px;
+    height: 48px;
+    padding: 15px;
+    top: 13%;
+    background: #fff;
+    box-shadow: 0 16px 24px rgba(37, 47, 137, 0.08);
+    border-radius: 50%;
+  }
+
+  .skill_h2 {
+    position: relative;
+    top: 20%;
+    left: 20px;
+  }
+
+  .stack_skills1 {
+    position: absolute;
+    bottom: 30px;
+    left: 20px;
+  }
+
+  .stack_skills2 {
+    position: absolute;
+    bottom: -20px;
+    right: 440px;
+  }
+
+  .stack_skills3 {
+    position: absolute;
+    bottom: -25px;
+    right: 40px;
+  }
+
+  .frame {
+    position: relative;
+    top: 15px;
+  }
+
+  .stack_skills1 ul li {
+    font-size: 11px;
+    text-align: center;
+    padding: 10px;
+    background: #fff;
+    border-radius: 8px;
+  }
+
+  .stack_skills2 ul li {
+    font-size: 11px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    padding: 10px;
+    background: #fff;
+    border-radius: 8px;
+  }
+
+  .stack_skills3 ul li {
+    font-size: 11px;
+    letter-spacing: 0.03em;
+    padding: 10px;
+    background: #fff;
+    margin: 0;
+    border-radius: 8px;
+  }
+
+  /* contact section */
+
+  #img_contact {
```

Comment:
The background image for the contact section is missing as per the Figma design. Kindly fix this issue as well 💯 

Your Design
<img width="1440" alt="Screenshot 2023-01-21 at 10 43 19 AM" src="https://user-images.githubusercontent.com/24832846/213844777-544cd079-adb0-42f8-a2c5-6430f3791ff1.png">

Figma Design
<img width="569" alt="Screenshot 2023-01-21 at 10 43 07 AM" src="https://user-images.githubusercontent.com/24832846/213844781-0006e504-cb34-4475-87f3-08d34861f744.png">